libraries/rush-lib/src/logic/RepoStateFile.ts

```diff
@@ -71,6 +78,13 @@ export class RepoStateFile {
     return this._preferredVersionsHash;
   }
 
+  /**
+   * The hash of all preferred versions at the end of the last update.
+   */
+  public get packageJsonInjectedDependenciesHash(): string | undefined {
+    return this._packageJsonInjectedDependenciesHash;
+  }
```

Comment:
Can you just make this a `public readonly` property?

Reply:
Same with other accessors of simple properties.

Reply:
Can not use readonly here.
`'readonly' modifier can only appear on a property declaration or index signature.ts(1024)`